src/UI/Launcher.module.css

```diff
@@ -1,6 +1,6 @@
 .launcher {
 	position: fixed;
-	z-index: 2000;
+	z-index: calc(var(--parley-index-prefix, 100) + 1);
```

Comment:
hoe kan deze z-index eerst hoger zijn dan die van `ReplyActions.module.css -> .footer` en nu lager?
Hoort ie niet in de zelfde volgorde te staan als eerst?

Reply:
Ja mijn verwarring kwam doordat ie dus eerst hoger was dan de ReplyActions en nu opeens lager.
Wat we willen is dat die achter de chat valt dus dan is lager idd beter (weet alleen niet waarom die voorheen met een hogere z-index nog steeds achter de chat viel 🤔 )

Reply:
> (weet alleen niet waarom die voorheen met een hogere z-index nog steeds achter de chat viel 🤔 )

Als je dit nog kan verklaren zou dat fijn zijn, anders mag je deze conversatie zelf resolven.